src/lib/components/csvImportBox.svelte

```diff
@@ -179,6 +185,22 @@
 
     $: isOpen = true;
     $: showCsvImportBox = $importItems.size > 0;
+
+    let showDetails = false;
+    let selectedErrors: string[] = [];
+
+    function openDetails(errors: string[] | undefined) {
+        selectedErrors = errors ?? [];
+        showDetails = true;
+    }
+
+    $: parsedErrors = selectedErrors.map((err) => {
+        try {
+            return JSON.stringify(JSON.parse(err), null, 2);
+        } catch {
+            return err;
+        }
+    });
```

Comment:
why reactive? lets move this to when the modal is shown so we don't update on every reactive trigger.